examples/ex1.cpp

```diff
@@ -139,27 +141,30 @@ int main(int argc, char *argv[])
    b->AddDomainIntegrator(new DomainLFIntegrator(one));
    b->Assemble();
 
+   // 7. Set MFEM config parameters from the command line options
    mesh->SetCurvature(1, false, -1, Ordering::byVDIM);
-   if (gpu) { config::Get().Cuda(true); }
-   if (pa)  { config::Get().PA(true);   }
+   config::SetCuda(cuda);
```

Comment:
These method names still aren't clear on what they do for users. 
Since the getter is called `usingCuda` maybe the setter should be called `useCuda`

Reply:
Done, turned the `Set` to `use` for the Cuda, Occa and PA.